.github/workflows/complement.yml

```diff
@@ -0,0 +1,36 @@
+# This task does not run complement tests, see tests.yaml instead.
+# This task does not build docker images for synapse for use on docker hub, see docker.yaml instead
+
+name: Store complement synapse image in ghcr.io
+on:
+  push:
+    branches: ["main", "develop"]
```

Comment:
Note that Synapse's primary branch is still called `master`

Reply:
Ah, yes; i was cribbing from another repo. Thanks for the catch.